doc/manuals/user/forbidden_characters.md

```diff
@@ -15,44 +15,39 @@ following characters are forbidden in any request:
 -   (
 -   )
 
-Any attempt of using them will result in a NGSI 400 Bad Request response
+Any attempt of using them will result in a 400 Bad Request response
 like this:
 
     {
-        "orionError": {
-            "code": "400",
-            "details": "Illegal value for JSON field",
-            "reasonPhrase": "Bad Request"
-        }
+        "error": "BadRequest",
+        "description": "Invalid characters in attribute type"
     }
 
 If your aplication needs to use these characteres, you should encode it
 using a scheme not including forbidden characters before sending the
-request to Orion (e.g. [URL
-encoding](http://www.degraeve.com/reference/urlencoding.php)).
+request to Orion. 
 
-There is another set of characters that requires special care from the 
-user perspective. Namely, the ones in the following list:
+[URL encoding](http://www.degraeve.com/reference/urlencoding.php)) is
+a valid way of encoding. However, we don't recommend its usage for
+fields that may appear in API URL (such as entity id or attribute names)
+due to it would need to encode the "%" character itself. For instance,
+if we would want to use "E<01>" as entity id, its URL encode would be:
+"E%3C01%3E".
 
--   #
```

Comment:
I have take the opportunity to remove also this part about #, ?, etc. It is redudant, as it is already addressed by section "Specific restrictions for ID fields".

NTC (informative)